pkgs/top-level/release-haskell.nix

```diff
@@ -301,28 +381,24 @@ let
           ];
         };
         constituents = [
```

Comment:
With `removePlatforms` the need to specify the platforms explicitly should be removed and we could use `accumulateDerivations` again here.

Reply:
Ah, good catch.  I fixed this in fb722f4.